### PR TITLE
docs: Make docs more concise; mention Fourmat

### PR DESCRIPTION
**Pull request by sloria**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,154 +1,22 @@
...
-# Good:
-from foo import (
-    bar, baz, qux,
-)
-
-# Bad:
-from foo import \
-    bar, baz, qux
-```
+We rely on [fourmat](https://github.com/4Catalyzer/fourmat) to automatically format and lint code according to our style guidelines. Below are additional guidelines that are not taken care of by `fourmat`.
```
Comment (taion) [inline README.md]:
```suggestion
We rely on [Fourmat](https://github.com/4Catalyzer/fourmat) to automatically format and lint code according to our style guidelines. Below are additional guidelines that are not taken care of by Fourmat.
```

File: README.md [hunk 2]
```diff
@@ -1,154 +1,22 @@
...
 
 ## Data types
 
 Prefer immutable data types over their mutable equivalents. Specifically, prefer `tuple` and `frozenset` over `list` and `set` respectively when the relevant collections will not be mutated. However, it is acceptable to use `list` over `tuple` for collections that semantically constitute homogeneous lists rather than heterogeneous tuples, though `tuple` may still be better when the list is short.
 
 ## Import styles
 
-If a module defines a homogeneous collection of items (e.g. `models`, `views` or `layers`), prefer importing the module over the individual items from that module.
+If a module defines a homogeneous collection of items (e.g. `models`, `views` or `layers`), prefer importing the module over the individual items from that module.
```
Comment (taion) [inline README.md]:
```suggestion
If a module defines a homogeneous collection of items (e.g. `models`, `views` or `layers`), prefer importing the module over the individual items from that module.
```

Comment (taion) [reply README.md]: huh?

Comment (taion) [reply README.md]: also this suggestion is probably wrong for flask-resty "resources"

File: README.md [hunk 3]
```diff
@@ -165,22 +33,18 @@ my_package/my_module.py
...
-my_dict['my_key'] = my_value
-raise ApiError(400, {'code': 'invalid_value'})
-```
-
 ## Section dividers
 
 It is acceptable to use the following section divider when it aids clarity. For example, this section divider can be useful after imports or between groups of related functions.
 
-```python
+```
```
Comment (taion) [inline README.md]: this is python :p

File: README.md [hunk 4]
```diff
@@ -165,22 +33,18 @@ my_package/my_module.py
...
-
-```python
-logger.info("this is a human-readable message")
-raise RuntimeError("something has gone wrong")
-
-my_dict['my_key'] = my_value
-raise ApiError(400, {'code': 'invalid_value'})
-```
-
 ## Section dividers
```
Comment (taion) [inline README.md]: i wouldn't be averse to getting rid of this peculiarity. it's a weird thing i picked up from someone at my last job, and i don't know that it's particularly helpful

**Review by taion — APPROVED** [review]

File: README.md (hunk 4 above)
Comment (itajaja) [reply README.md]: lol yeah. not a big fan too

File: README.md (hunk 2 above)
Comment (sloria) [reply README.md]: Hm, dunno what that space is about..probably some weirdness from importing then exporting from Notion.

Should we just remove this section?

File: README.md (hunk 4 above)
Comment (sloria) [reply README.md]: I don't hate them =) . But I can remove this section.

**Comment by sloria** [comment]
After removing the stuff that's either 1) taken care of Fourmat or 2) we no longer prescribe, there's only 1 section left 😄

**Pull request merged**